eng/init-source-only.proj

```diff
@@ -33,6 +33,73 @@
           WorkingDirectory="$(PreviouslySourceBuiltPackagesPath)" />
   </Target>
 
+  <ItemGroup>
+    <SharedComponentsPrereqsArchivePathItem Include="$(ExternalTarballsDir)$(SourceBuiltSharedComponentsTarballName).*$(ArchiveExtension)" />
+  </ItemGroup>
+  <PropertyGroup>
+    <SharedComponentsPrereqsArchivePath>@(SharedComponentsPrereqsArchivePathItem)</SharedComponentsPrereqsArchivePath>
```

Comment:
Thought on using the file suffix versus path?  This would be more consistent with SourceBuiltPrebuiltsTarballFile.

Reply:
I am simply referring to updating the property name to be `SharedComponentsPrereqsArchiveFile`

Reply:
Fixed in https://github.com/dotnet/dotnet/pull/1502/commits/3c11f4c04e79cb871ac7c7a1d79f58c51066f38e